ui/src/app/job-list/table/table.component.ts

```diff
@@ -249,4 +261,59 @@ export class JobsTableComponent implements OnInit {
       }
     }
   }
+
+  openbulkEditDialog(): void {
+    this.resetBulkLabelFieldDefaults();
+    // figure out default values for bulk fields; if all jobs have the same label value,
+    // set that to the default; otherwise, set default to boolean false
+    for (let job of this.selection.selected) {
+      for (let bulkFieldItem of this.bulkLabelFields) {
+        const label = bulkFieldItem.displayField.field.replace('labels.', '');
+        const jobLabelValue = job.labels[label] || '';
+        if (bulkFieldItem.default === null) {
+          bulkFieldItem.default = jobLabelValue;
+        } else if (bulkFieldItem.default !== false && bulkFieldItem.default !== jobLabelValue) {
+          bulkFieldItem.default = false;
+        }
+      }
+    }
+    let dialogRef = this.bulkEditDialog.open(JobsBulkEditComponent, {
+      disableClose: true,
```

Comment:
Is there another way to cancel the bulk edit? I don't see this in the dialog

Reply:
There is a 'Cancel' and an 'Apply' button